battleship.py

```diff
@@ -109,7 +109,15 @@ def checkShip(grid, ship):
 Returns: 2D list of ints
 '''
 def addShips(grid, numShips):
-    return
+    board=grid
+    count=0
+    while count < numShips:
+        ship=createShip()
+        if checkShip(board, ship) == True:

```

Comment:
Good job @Vedvundmsit 
In the 116th line, you don't explicitly need to check the condition with True. You can leave like if checkShip(board, ship): This will work too.